### PR TITLE
Conditionalize the Python 2 subpackage and don't build it on EL > 7 and Fedora > 28

### PR DESCRIPTION
**Pull request by hroncok**
- Use bcond for with python3, allow it on RHEL > 7
- This reverts commit 243b84bb9629a09e77dcb9f8fb87414f5b679c33
- See https://src.fedoraproject.org/rpms/python-blivet/pull-request/2

### CODE REVIEW

File: python-blivet.spec [hunk 1]
```diff
@@ -76,13 +85,21 @@ Requires: systemd-udev
 Requires: %{realname}-data = %{epoch}:%{version}-%{release}
 
 Obsoletes: blivet-data < 1:2.0.0
+
+%if %{without python2}
+# the version here needs to be updated to actual version when this happened
+Obsoletes: python2-blivet < 1:2.0.2-2
```
Comment (hroncok) [inline python-blivet.spec]: Note that in my original PR this was 3.0.0-0.6, here the version in the spec is 2.0.2 but that is nonsense.

Comment (hroncok) [reply python-blivet.spec]: Can be changed to `< %{version}-%{release}` - that works but is hostile towards potential return of the python2 package (in a module, etc.). I'd do that if such scenario is unlikely, let me know.

Comment (vojtechtrefny) [reply python-blivet.spec]: We have a really complicated system of branches -- the right version is only in spec on `3.0-release` and we merge `3.0-devel` to `3.0-release` before every upstream release. Thanks to this we can merge bugfixes from `2.1-devel` to `2.2-devel` and `3.0-devel` without opening new pull requests or creating separate patches. As I said, it's very complicated and you can just ignore the version written here.

tl;dr version -- the version is ok, thank you for the changes.

**Comment by hroncok** [comment]
This a bit time sensitive for us. Is there anything I can do to speed things up? Thanks.

**Comment by hroncok** [comment]
Thanks
